docs/content/guides/recipes/terraform/howto-custom-provider/index.md

```diff
@@ -43,7 +43,7 @@ Create a Bicep file `env.bicep` with the secretStore resource:
 
 `recipeConfig/terraform/providers` allows you to setup configurations for one or multiple Terraform Providers. For more information refer to the [Radius Environment schema]({{< ref environment-schema >}}) page.
 
-In your `env.bicep` file add an Environment resource, along with Recipe configuration which leverages properties from the previously defined secret store. In this example you're also passing in `host` as an environment variable to highlight use cases where, depending on provider configuration requirements, users can pass environment variables to the Terraform recipes runtime.
+In your `env.bicep` file add an Environment resource, along with Recipe configuration which leverages properties from the previously defined secret store. In this example you're also passing in `host` and `port` as environment variables to highlight use cases where, depending on provider configuration requirements, users can pass environment variables as plain text and as secret values to the Terraform recipes runtime.
```

Comment:
would this be more accurate/clear?
```suggestion
In your `env.bicep` file add an Environment resource, along with Recipe configuration which leverages properties from the previously defined secret store. In this example you're also passing in `host` and `port` as environment variables to highlight use cases where, depending on provider configuration requirements, users can pass environment variables as plain text and as `envSecrets` values to the Terraform recipes runtime.
```

Reply:
updated to `secret values in 'envSecrets' block` as discussed